data/en/filewrite.json

```diff
@@ -6,7 +6,7 @@
 	"related":[],
 	"description":" If you specify a file path, writes the entire content to the specified on-disk or in-memory file. If you specify a file object, writes text or binary data to the file object.",
 	"params": [
-		{"name":"filePath","description":"No Help Available","required":true,"default":"","type":"String","values":[]},
+		{"name":"filePath or fileObject","description":"No Help Available","required":true,"default":"","type":"String","values":[]},
 		{"name":"data","description":"No Help Available","required":true,"default":"","type":"String","values":[]},
```

Comment:
"No Help Available" should be solved

Reply:
What if we just call the argument `file` and then in the description note that it can be a file path string or a file object, and set the `type` to `any`, for example:

`{"name":"file","description":"A file object or a file system path string.","required":true,"default":"","type":"any","values":[]},`

Reply:
You would also want to update the syntax to be:

`"syntax":"fileWrite(file, data [, charset])"`

Reply:
That sounds good to me :+1: